tests/e2e/README.md

```diff
@@ -87,6 +87,11 @@ You can build and run this or any single test manually with the same options as
 
 ## Developer process
 
+### Cluster in same local network
+In order to talk to istio ingress, we use the ingress IP by default. If your
+cluster is on the same local network and cannot provide external IP, use the `--use-local-cluster` flag.
+In that case, the framework will not create a LoadBalancer and talk directly to the Pod running istio-ingress.
+
```

Comment:
This looks good but I would suggest "... cannot provide external IP (for example, minikube), use ...".

I also think you should remove the this sentence from the top of the README: "We recommend GKE cluster although miniKube works for some people too". It works fine for minikube.